src/core/thread/network_data.cpp

```diff
@@ -658,6 +658,136 @@ ThreadError NetworkData::SendServerDataNotification(uint16_t aRloc16)
     return error;
 }
 
+ThreadError NetworkData::SendGetCommissioningRequest(const uint8_t *aTlvs, uint8_t aLength)
```

Comment:
Yes, commisionser sends it, but commissioning data is part of network data. So I put it here.
I can move it.